stories/components/InputCollection.md

```diff
@@ -0,0 +1,48 @@
+# InputCollection
```

Comment:
This markdown file is not connected to storybook.
You can reference `stories/components/Carousel/Carousel.stories.tsx`